test/Util/TestUtilCasacore.cc

```diff
@@ -233,15 +233,21 @@ TEST_F(GetResolvedFilenameTest, ResolvesNormalFile) {
 	EXPECT_TRUE(message.empty());
 }
 
-/* FAILING
-* expected output -> /tmp/test_symlink/subdir/symlink.txt
-* actual output -> /tmp/test_symlink/subdir/real.txt
-* the GetResolvedFilename function resolves the filename to the real file instead
-* where it actualy should resolve to the file through the symlink
-*
-* path.resolvedName() -> it is a known casacore issue that it fails to resolve symlinks
-* path.expandedName() -> if resolvedName() fails it tries expandedName() which returns the expanded name but neccessarily the resolved path
-*/
+TEST_F(GetResolvedFilenameTest, ResolvesToSameDirectory) {
+    auto pwd = TestRoot();
+	fs::path dir1 = temp_dir / "subdir";
+	fs::create_directory(dir1);
+    fs::path dir2 = temp_dir / "" / "subdir";
+    fs::create_directory(dir2);
```

Comment:
This is not necessary -- this just depends on fs::path behaviour, which we aren't testing.